### ISSUE TITLE
The query run error results

### ISSUE THREAD

**Issue by idiomaticrefactoring** [PyGithub/PyGithub#1773]
The link, https://pygithub.readthedocs.io/en/latest/examples/MainClass.html#search-repositories-based-on-number-of-issues-with-good-first-issue, says that it can search  issues with good-first-issue.

However, the result seems wrong.

For example, there is project whose number of good first issue larger than 400, such as elastic/kibana.

And when I search using the following code : 
     query = 'good-first-issue:>0' or query = 'good-first-issue:>1'
    repositories = g.search_repositories(query=query)

the number of project ">0" is less than the number of project ">1"

**Therefore, is there some query that can search projects whose issues with different labels?**

**Comment by johnsc1** [comment]
Hello, a group of student developers I am a part of are claiming this issue, and currently working toward a solution to which handles different issue labels that have the same purpose.

**Comment by johnsc1** [comment]
We have tested several different queries, including "good-first-issue". We discovered that the query "good-first-issues:>x" where "issues" is plural and x is some number (such as 3 in the[ PyGithub example](https://pygithub.readthedocs.io/en/latest/examples/MainClass.html#search-repositories-based-on-number-of-issues-with-good-first-issue)), seems to behave differently than other queries. The other queries we tested seemed to only return results which exactly matched the issue label in the query string. 

When we tested "good-first-issues:>3" as a Github search online, we discovered that unlike other queries, which returned both issues and repositories, "good-first-issues" returns no issues and many repositories. Unlike the other queries, which returned only repositories with issue labels which exactly match the query string, "good-first-issues" was returning repositories with issue labels that both did and did not match exactly, such as "good-first-issues" and "good first issue". 

Is a reasonable exception that "good-first-issues:>x" is a special query?
